cmd/cli/cmd/action/wait.go

```diff
@@ -0,0 +1,42 @@
+package action
+
+import (
+	"os"
+	"time"
+
+	"capact.io/capact/internal/cli"
+	"capact.io/capact/internal/cli/action"
+	"capact.io/capact/internal/cli/client"
+	"capact.io/capact/internal/cli/heredoc"
+
+	"github.com/spf13/cobra"
+)
+
+// NewWait returns a new cobra.Command for waiting for a given Action's condition.
+func NewWait() *cobra.Command {
+	var opts action.WaitOptions
+
+	cmd := &cobra.Command{
+		Use:   "wait ACTION",
+		Short: "Wait for a specific condition on a given Action",
```

Comment:
Shouldn't be "of a given Action"? 🤔  https://www.macmillanthesaurus.com/topics/the-condition-of-something

```suggestion
		Short: "Wait for a specific condition of a given Action",
```